lib/src/test/java/uk/gov/di/ipv/core/library/domain/gpg45/Gpg45ProfileEvaluatorTest.java

```diff
@@ -419,78 +423,143 @@ private void setupMockContraIndicatorTreatmentConfig() throws ConfigException {
+            private String code;
+            private List<String> mitigations;
+
+            TestContraIndicators(String code, List<String> mitigations) {
+                this.code = code;
+                this.mitigations = mitigations;
+            }
+
+            TestContraIndicators(String code) {
+                this(code, List.of());
+            }
+        }
+
+        private ContraIndications makeContraIndications(
+                TestContraIndicators... testContraIndicators) {
             return ContraIndications.builder()
                     .contraIndicators(
-                            Arrays.asList(contraIndicatorCodes).stream()
+                            Arrays.stream(testContraIndicators)
                                     .collect(
                                             Collectors.toMap(
-                                                    code -> code,
-                                                    code ->
+                                                    testContraIndicator -> testContraIndicator.code,
+                                                    testContraIndicator ->
                                                             ContraIndicator.builder()
-                                                                    .code(code)
+                                                                    .code(testContraIndicator.code)
                                                                     .issuanceDate(Instant.now())
+                                                                    .mitigations(
+                                                                            testContraIndicator
+                                                                                    .mitigations
+                                                                                    .stream()
+                                                                                    .map(
+                                                                                            mitigation ->
+                                                                                                    Mitigation
+                                                                                                            .builder()
+                                                                                                            .code(
+                                                                                                                    mitigation)
+                                                                                                            .build())
+                                                                                    .collect(
+                                                                                            Collectors
+                                                                                                    .toList()))
```

Comment:
Many layers of indentation here, only test code but suspect this could be more readable

Reply:
Definitely. Split out into smaller methods.